build-scripts/ubuntu-1604/build-3rd-parties.sh

```diff
@@ -51,3 +51,4 @@ build_from_pypi intervaltree 2.1.0
 build_from_pypi ujson 1.33
 build_from_pypi portalocker 0.5.7
 build_from_pypi sortedcontainers 1.5.7
+build_from_pypi msgpack-python 0.4.8
```

Comment:
The package already exists in the Canonical archive can we use `0.4.6`?
```
python3-msgpack/xenial 0.4.6-1build1 amd64
  Python 3 implementation of MessagePack format
```
If so please add altering this dependency in `prepare-package.sh`.
Like for other packages:
```
sed -i "s~ujson==1.33~ujson==1.33-1build1~" "$repo/setup.py"
sed -i "s~prompt_toolkit==0.57~prompt_toolkit==0.57-1~" "$repo/setup.py"
```

And remove this line.